boards/nxp/mimx8mp_evk/Kconfig.mimx8mp_evk

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_MIMX8MP_EVK
+	select SOC_MIMX8MP_ADSP if BOARD_MIMX8MP_EVK_MIMX8ML8_ADSP
+	select SOC_PART_NUMBER_MIMX8ML8DVNLZ
+	select SOC_PART_NUMBER_MIMX8ML8CVNKZ
```

Comment:
how can this have 2 part numbers?

Reply:
Well.. good question - I don't know :(
This is how is defined in hal - https://github.com/zephyrproject-rtos/hal_nxp/blob/master/mcux/mcux-sdk/devices/MIMX8ML8/Kconfig#L22 and I enabled the DSP from those parts.

It's ok thata re only 2 - we could have more - https://www.nxp.com/products/processors-and-microcontrollers/arm-processors/i-mx-applications-processors/i-mx-8-applications-processors/i-mx-8m-plus-arm-cortex-a53-machine-learning-vision-multimedia-and-industrial-iot:IMX8MPLUS:

Reply:
also see here: https://raw.githubusercontent.com/zephyrproject-rtos/hal_nxp/master/mcux/mcux-sdk/devices/MIMX8ML8/MIMX8ML8_dsp.h
```
/*
** ###################################################################
**     Processors:          MIMX8ML8CVNKZ_dsp
**                          MIMX8ML8DVNLZ_dsp
**
**     Compiler:            XCC Compiler
**     Reference manual:    IMX8MPRM, Rev.D, 12/2020
**     Version:             rev. 5.0, 2021-03-01
**     Build:               b230302
**
```

Reply:
this cannot have 2 part numbers, select the correct part number

Reply:
@nordicjm I think it relates to how the chip is certified. @iuliana-prodan we can go with just the first part number in the list for now.

@nordicjm Is there a limitation so that we cannot support two part numbers in the same time?